scripts/validate.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import os
+from github import Github
+
+def check_is_verified(token, repository, ref):
```

Comment:
It'd be good to add some testing infrastructure with `pytest` for this given there's a few different states it could be in, worth doing that before we get too many cases - probably a separate GitHub Action for that?